iohub/cli/cli.py

```diff
@@ -117,3 +119,32 @@ def convert(
         chunks=chunks,
     )
     converter.run(check_image=check_image)
+
+
+@cli.command()
+@click.help_option("-h", "--help")
+@input_position_dirpaths()
+@click.option(
+   "--x-scale",
+   "-x",
+    required=False,
+    type=float,
+    help="New x scale",
+)
+@click.option(
+   "--y-scale",
+   "-y",
+    required=False,
+    type=float,
+    help="New y scale",
+)
+@click.option(
+   "--z-scale",
+   "-z",
+    required=False,
+    type=float,
+    help="New z scale",
+)
+def update_scale_metadata(input_position_dirpaths, z_scale, y_scale, x_scale):
+    """Update scale metadata in OME-Zarr datasets"""
```

Comment:
I will suggest adding an example call like `>> iohub update-scale-metadata -i <input> ... etc` here. 

Double check that it works.